### PR TITLE
LEX-504 Eliminate the extra CaptureIdentity event and capture identit…

### PR DESCRIPTION
**Pull request by ynx**
…y during the Content View event.

### CODE REVIEW

File: acquia_lift_profiles/js/acquia_lift_profiles.js [hunk 1]
```diff
@@ -232,12 +198,20 @@ var _tcwq = _tcwq || [];
...
           _tcwq.push(['onLoad', segmentCallback]);
-          _tcaq.push(['captureView', 'Content View', pageInfo]);
 
-          if(settings.acquia_lift_profiles.hasOwnProperty('identity')) {
-            pushCaptureIdentity(settings.acquia_lift_profiles.identity, settings.acquia_lift_profiles.identityType);
+          // Capture view.
+          var captureViewEvent = ['captureView', 'Content View', pageInfo];
+
+          // Capture identity in addition, if applicable.
+          if(!identityCaptured && settings.acquia_lift_profiles.hasOwnProperty('identity')) {
```
Comment (eshtadc) [inline acquia_lift_profiles/js/acquia_lift_profiles.js:206]: Do you also have to check settings.acquia_lift_profiles.captureIdentity like you do below on line 258.

File: acquia_lift_profiles/qunit/tests.js [hunk 2]
```diff
@@ -82,12 +82,14 @@ QUnit.module("Acquia Lift Profiles", {
         }, 1000);
       }
     };
+  },
+  teardown: function() {
+    Drupal.acquia_lift_profiles.resetAll();
```
Comment (eshtadc) [inline acquia_lift_profiles/qunit/tests.js:87]: Yay :)

File: acquia_lift_profiles/js/acquia_lift_profiles.js (hunk 1 above)
Comment (ynx) [inline acquia_lift_profiles/js/acquia_lift_profiles.js:206]: Sharp observation Lisa ... 

"captureIdentity" flag is set in Acquia Lift Profiles page and is the switch only for whether or not to capture user_login and user_register. The variable name is misleading as it doesn't control the query string parameters capture identity here.

Comment (eshtadc) [inline acquia_lift_profiles/js/acquia_lift_profiles.js:206]: Oooh - I get it.  OK - figured it was something like that.  👍

**Pull request merged**